src/test/java/api/loans/DueDateNotRealTimeScheduledNoticesProcessingTests.java

```diff
@@ -37,12 +35,19 @@
 import io.vertx.core.json.JsonObject;
 
 public class DueDateNotRealTimeScheduledNoticesProcessingTests extends APITests {
+
+  private final static UUID TEMPLATE_ID = UUID.randomUUID();
+
+  @Before
+  public void setUp() {
+    templateClient.create(new JsonObject().put("id", TEMPLATE_ID.toString()));
```

Comment:
Maybe a method could be extracted here to better describe the intent.

```suggestion
    templateClient.create(dummyNoticeTemplate());
```